README_CN.md

```diff
@@ -150,9 +158,9 @@ $ yomo dev
 
 ## 📚 Documentation
 
-+ `YoMo-Source`: [yomo.run/source](https://docs.yomo.run/source)
-+ `YoMo-Stream-Function` (formerly flow): [yomo.run/stream-function](https://docs.yomo.run/stream-function)
-+ `YoMo-Zipper`: [yomo.run/zipper](https://docs.yomo.run/zipper)
++ `YoMo-Source`: [docs.yomo.run/source](https://docs.yomo.run/source)
++ `YoMo-Stream-Function`: [docs.yomo.run/stream-function](https://docs.yomo.run/stream-function)
```

Comment:
should be [docs.yomo.run/stream-fn](https://docs.yomo.run/stream-fn)

Reply:
done